config/database.yml

```
@@ -1,25 +1,20 @@
-# SQLite. Versions 3.8.0 and up are supported.
-#   gem install sqlite3
-#
-#   Ensure the SQLite 3 gem is defined in your Gemfile
-#   gem 'sqlite3'
-#
 default: &default
-  adapter: sqlite3
+  adapter: postgresql
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   timeout: 5000
 
 development:
   <<: *default
-  database: db/development.sqlite3
+  database: db/task_list_development
```

Nit:
看到 DB 名字前有多加應該是不必要的前綴 `db/` ，建議移除：
```suggestion
  database: task_list_development
```